src/sections/Kanvas/kanvas-icon.svg

```diff
@@ -11,10 +11,12 @@
       }
     </style>
   </defs>
+  <g transform="scale(1)">
```

Comment:
Instead of using the transform property in the SVG, which can cause sizing issues when the logo is used in different contexts, could you explore alternative ways to increase the size of the Kanvas logo? This will help ensure it maintains quality and consistent dimensions across various placements.

Reply:
@NishantSinghhhhh Thanks for the feedback.